src/ServerScriptService/GravityController/GravityController/Collider.lua

```diff
@@ -158,11 +162,15 @@ function ColliderClass:Update(force, cframe)
 end
 
 function ColliderClass:IsGrounded(isJumpCheck)
-	local parts = isJumpCheck and self.JumpDetector:GetTouchingParts() or self._floorTouchingParts
-	for _, part in pairs(parts) do
-		if not part:IsDescendantOf(self.Controller.Character) and part.CanCollide then
-			return true
+	if isJumpCheck then
```

Comment:
it wasn't deemed necessary to add a cache for the jumpCheck since it's on-demand rather than per-frame